data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/export/DataFileScheduler.java

```diff
@@ -31,8 +31,15 @@ public class DataFileScheduler implements Runnable {
 
     private final AtomicInteger numOfWorkers = new AtomicInteger(0);
 
-    private static final int MAX_JOB_COUNT = 2;
-    private static final int DEFAULT_LEASE_INTERVAL_MILLIS = 30_000;
+    /**
+     * Maximum concurrent data loader per node
+     */
+    private static final int MAX_JOB_COUNT = 4;
+
+    /**
+     * Default interval to acquire a lease from coordination store
+     */
+    private static final int DEFAULT_LEASE_INTERVAL_MILLIS = 15_000;
```

Comment:
Thanks Taylor, I do have those in my code at the very beginning but I removed it, like at first 15 seconds, then 30/60 seconds if no lease taken. Reset to 15 seconds if lease was taken.  Set it to 15 seconds minimum is so that other nodes can also take leases and the job can be evenly distributed.   The main backoff I put is the GetRecords call (which is to check the number of records/bytes returned), but it actually added complexity and increased latency.  So there is a trade off.  

But this is to say it still can be added in the future once we have done more testing.